### DISCUSSION TITLE
Wrong answers from correlated methods applied to centrosymmetric molecules

### DISCUSSION THREAD

**Discussion by SylviaJoy1** [pyscf/pyscf#2633] [Q&A]
I can match literature values of Ne and Ar RHF total energies, but some correlated methods using PySCF (v2.7.0) give incorrect answers for Ne and Ar.

~~Using PySCF's ADC module, I am able to reproduce [literature EA-ADC(2) values](https://doi.org/10.1063/1.5131771) for He, Be, CO, and N2, but not for Ne and Ar. 
/~~

The DOCI PySCF extension does not match [literature values for Ne](https://doi.org/10.1063/1.4904384). Increasing the number of roots requested does not fix the issue.

I coded pCCD by making minor modifications to PySCF's CCSD code. 
For Ne, Ar, and CH4, my results do not match pCCD results from the [PyBEST package](http://fizyka.umk.pl/~pybest/pybest-v2.0.0/index.html). 

I get different pCCD and DOCI results depending on whether symmetry is turned on or off for these molecules. As expected, DOCI and pCCD give ground state results that are very close to each other.

The He atom does not experience any of these problems, nor does H2O2. 
CCSD in PySCF does not give any of these problems for Ne or Ar.

~~When I use a PySCF-based EE-ADC(2) code from the Sokolov group, I obtain incorrect excitation energies for Ne but not for molecules of lower symmetry.~~

**Comment by tberkel** [comment]
Hi @SylviaJoy1 thanks for reporting this. Just to confirm, in light of your edits, there are now two issues (and a related third):
1. DOCI extension does not match literature for Ne atom (for both ground and excited states?)
2. Your pCCD does not match ground state results
3. Both DOCI and pCCD results depend on whether symmetry is on or off

Can you confirm? The pCCD one might be especially hard to debug since it's your own code, but the CCSD upon which it's based does not exhibit these issues. My intuition says that something about the initial guess is at the heart of all these issues ...
